File: src/main/java/no/nav/veilarbveileder/config/EnvironmentProperties.java
package no.nav.veilarbveileder.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Getter
@Setter
@ConfigurationProperties(prefix = "app.env")
public class EnvironmentProperties {
    private String openAmDiscoveryUrl;

    private String veilarbloginOpenAmClientId;

    private String naisStsDiscoveryUrl;

    private String naisAadDiscoveryUrl;

    private String naisAadClientId;

    private String abacVeilarbUrl;

    private String norg2Url;

    private String unleashUrl;

}

File: src/main/java/no/nav/veilarbveileder/config/EnvironmentProperties.kt
package no.nav.veilarbveileder.config

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "app.env")
data class EnvironmentProperties (
    val naisAadDiscoveryUrl: String,
    val naisAadClientId: String,
    val norg2Url: String,
    val poaoTilgangUrl: String,
    val poaoTilgangScope: String,
    val nomApiUrl: String,
    val nomApiScope: String,
    val microsoftGraphUri: String,
    val microsoftGraphScope: String
)
